### PR TITLE
Refactor: Slow image load /210825

### PR DESCRIPTION
**Pull request by mikaell**
This PR fixes a bug: Showing of Chromograph Images was very slow. Not it is just slow.

This PR introduces several marginal gains:
* Use fragments for construction of larger DOM elements
* Usage of `cloneNode()` instead of `document.createElementNS(...);` may have some 
* Set constants outside loop-conditions
* etc

**How to test**:
1. how to test it, possibly with real cases/data
Install branch. Visit case page and click *Images NA12882*. If configured Chromograph images will show and hopefully the user will experience a small improvement of the loading time.

**Expected outcome**:
The functionality should be working. Small but noticeable improvement in load times for the accordion element to expand/contract.

*No timed tests have been carried out. This has been done purely by feel.*

image

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Add extra Alamut links higher up on variant pages
 - Improve UX for phenotypes in case page
 - Filter and export of STR variants
+- Imporve Javascript performance for displaying images
 - Update look of variants page navigation buttons
+
```
Comment (northwestwitch) [inline CHANGELOG.md:26]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,7 +21,9 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Add extra Alamut links higher up on variant pages
 - Improve UX for phenotypes in case page
 - Filter and export of STR variants
+- Imporve Javascript performance for displaying images
```
Comment (northwestwitch) [inline CHANGELOG.md:24]:
```suggestion
- Improve Javascript performance for displaying Chromograph images
```

**Review by northwestwitch — APPROVED** [review]
It looks good. I wasn't even aware that we had a performance problem when loading the chromograph images. I can't really see a difference locally with the demo case. Tested on stage with this case:https://scout-stage.scilifelab.se/cust000/SWEDAC-2019-HG-05-Trio-NovaSeq and feels a bit faster. Tried also to zoom in and zoom out, just in case and the chromosome image behave.

👍🏻

**Comment by mikaell** [comment]
Will merge after tests pass 🙂

**Pull request merged**